app/domain/WalletTransaction.js

```diff
@@ -41,4 +50,12 @@ export default class WalletTransaction {
     return assuranceLevels.HIGH;
   }
 
+  getState(): TransactionState {
+    switch(this.condition) {
+      case 'CPtxApplying': return 'pending';
```

Comment:
Well actually, if you ever have a short doc over this, meaning of `pending` would hardly have one-sentence explanation, because incoming transaction with `CPtxNotTracked` state could be not yet in blockchain as well. But yep, for now it's the closest approximation.

Reply:
Ok thanks for the explanation @Martoon-00 if you agree that this is a good-enough abstraction for end-users then i will continue like that?!

Reply:
Sure :wink: